cloud-nio/cloud-nio-impl-ftp/src/main/scala/cloud/nio/impl/ftp/FtpCloudNioFileProvider.scala

```diff
@@ -5,7 +5,6 @@ import java.nio.channels.{Channels, ReadableByteChannel, WritableByteChannel}
 import java.nio.file.FileAlreadyExistsException
 
 import cats.effect.IO
-import cats.syntax.functor._
```

Comment:
sbt and IntelliJ are of differing opinions as to whether this and the following two functor imports are required. sbt thinks they are, which is one reason all the builds failed. 🙂

Reply:
🤦‍♂️